src/components/AddTerm/AddTerm.mobile.tsx

```diff
@@ -3,40 +3,56 @@ import styled, { StyledFunction } from 'styled-components';
 
 import { withProps } from '../../utils/styleUtils';
 
-export namespace AddTerm {
-  export interface Props {
-    defClassName: string;
-  }
-}
+// export namespace AddTerm {
+//   export interface Props {
+//     defClassName: string;
+//   }
+// }
 
 const defClassName = 'def';
-const StyledAddTerm = withProps<AddTerm.Props>()(styled.div)`
+const StyledAddTerm = styled.div`
+  padding : 10px;  
   input {
     width: 100%;
     height: 40px;
+    border : 1px solid black;
   }
   .term {
-
   }
-  ${props => props.defClassName}
+  button {
+    width : 100%;
+    border : none;
+    height : 40px;
+    outline : none;
+    cursor : pointer;
+  }
 `;
 
+const _renderUsage = (usages, handleClickAddUsage) => {
+  return (
+    <div>
+      <p>쓰임새</p>
+      {usages.map ( usage => {
```

Comment:
Either write in one of the forms.

```javascript
{usages.map(usage => (
  <input 
     defaultValue={usage} 
     placeholder="말의 쓰임새를 써주세요(ex: 실화니;이거실화임?)"/>
);}
```

Or better be sit in function
```javascript
{renderUsages()}
```
and this would be invoked by
```javascript
<div>
  <p>쓰임새</p>
  {renderUsages()}
</div>
```